lib/utils/EventLogRiemannClient.js

```diff
@@ -52,7 +52,7 @@ class EventLogRiemannClient {
 
   handleEvent(message, data) {
     try {
-      if (data.event) {
+      if (data.event && _.get(data, 'event.response.status') != CONST.HTTP_STATUS_CODE.BAD_REQUEST) {
```

Comment:
Is it not the same as `data.event.response.status`? 
But may be _.get is better way of writing code??

Reply:
Actually _.get handles all null cases implicitly . Hence no additional handling required by us.

Reply:
ok.. understood.